### PR TITLE
ME committee scraper

### PR DESCRIPTION
**Pull request by a-cat-named-snowball**

### CODE REVIEW

File: scrapers_next/me/committies.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        # Then remove all empty strings from the list
+        while "" in data_blocks:
+            data_blocks.remove("")
+
+        # data_blocks is now a nice list containing the 3 or 4 pieces of data
+        # If it does not contain a role, then it is in this format: [name,_,_]
+        # If it contains a role, then it is in this format: [role,name,_,_]
+        role = "Member"
+        name = None
+        match len(data_blocks):
```
Comment (johnseekins) [inline scrapers_next/me/committies.py:224]: match/case isn't supported in python 3.9 (which is what openstates-scrapers are pinned to/actually running in prod)

File: scrapers_next/me/committies.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+        # - Member role (optional)
+        # - Member name
+        # - Party-district
+        # - The string "View Profile"
+        # The individual blocks of data do not contain double spaces, so we can
+        # split by "  ".
+        data_blocks = text.split("  ")
+
+        # Then remove all empty strings from the list
+        while "" in data_blocks:
```
Comment (johnseekins) [inline scrapers_next/me/committies.py:216]: `data_blocks = [b for b in data_blocks if b]` seems much cleaner.

**Comment by johnseekins** [comment]
Along with other comments, please check on the linting errors.

File: scrapers_next/me/committies.py (hunk 2 above)
Comment (a-cat-named-snowball) [reply scrapers_next/me/committies.py]: It is a lot cleaner this way, thank you. Fixed with https://github.com/openstates/openstates-scrapers/pull/4425/commits/3e348341945ab1fee61cda77bf06f7099b8e947f

**Review by johnseekins — APPROVED** [review]

**Review by chrisyamas — APPROVED** [review]
Data looks good! Another great scraper.

**Pull request merged**